google_compute_engine/accounts/tests/accounts_utils_test.py

```diff
@@ -148,6 +148,28 @@ def testCreateSudoersGroupError(
     ]
     self.assertEqual(mocks.mock_calls, expected_calls)
 
+#  @mock.patch('google_compute_engine.accounts.accounts_utils.open')
+  @mock.patch('google_compute_engine.accounts.accounts_utils.os.path.exists')
+  def testCreateSudoersGroupWriteError(self, mock_exists):
+    mock_open = mock.mock_open()
+    mocks = mock.Mock()
+    mocks.attach_mock(mock_exists, 'exists')
+    mocks.attach_mock(self.mock_utils._GetGroup, 'group')
+    mocks.attach_mock(self.mock_logger, 'logger')
+    self.mock_utils._GetGroup.return_value = True
+    mock_exists.return_value = False
+    mock_open.side_effect = IOError()
+    accounts_utils.AccountsUtils._CreateSudoersGroup(self.mock_utils)
```

Comment:
Please move 162 to 163 (so that the whitespace is before the function call logic).